tests/element_test_vectors.rs

```diff
@@ -13,56 +13,6 @@ use test_generator::test_resources;
 /// Buffer size for our writing tests around round-tripping
```

Comment:
PR tour: this file has been heavily refactored so it can run the same tests for both `IonCElementReader` and `NativeElementReader`. The logic is unchanged.

Reply:
That's very reassuring given that the diff is so large GitHub won't load it by default.